src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+
+/**
+ * Custom exception class for Duke application.
+ * DukeException is thrown when there are errors or exceptions specific to the Duke application.
+ */
+public class DukeException extends Exception {
```

Comment:
Hi Andre, code looks hella clean!

I think it will also be nice to segment your code into packages for nicer file structure

Something similar to this:
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/122634188/760fede6-814b-417f-a278-90264955cc23)